src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -130,24 +129,42 @@ export interface IScatterPlotSampleData {
     profiledMutations?:boolean;
     mutations: AnnotatedMutation[];
     copyNumberAlterations: AnnotatedNumericGeneMolecularData[];
+    xThresholdType?: '>'|'<'|undefined; // threshold symbol for x coordinate in scatter plot
+    yThresholdType?: '>'|'<'|undefined; // threshold symbol for y coordinate in scatter plot
+    thresholdType?: '>'|'<'|undefined; // threshold symbol for continuous axis in box-scatter and waterfall plot
 }
 
-export interface IScatterPlotData extends IScatterPlotSampleData, IBaseScatterPlotData {};
-
-export interface IBoxScatterPlotPoint extends IScatterPlotSampleData {
+export interface IBoxScatterPlotPoint extends IPlotSampleData {
     category:string;
     value:number;
     jitter:number;
 }
 
+export interface IWaterfallPlotSampleData extends IPlotSampleData {
```

Comment:
I don't understand what this interface is. the `SampleData` indicates to me you're going to have one of these per sample, as with the `SampleData` interfaces above, but this seems to hold plot-level attributes. Do you need to store sortOrder and pivotThreshold on a per-sample basis?

Reply:
I have removed this interface.